ui/webui/test/check-basic

```diff
@@ -35,15 +35,10 @@ class TestBasic(MachineCase):
 
         b.open("/cockpit/@localhost/anaconda-webui/index.html")
 
-        b.click('#continue-btn')
-        b.expect_load()
-
-        for page in ['keyboard', 'language', 'timedate', 'installation-source', 'software-selection', 'installation-destination', 'network-hostname', 'root-account', 'user-account']:
+        for page in ['installation-language', 'installation-destination', 'review-configuration', 'installation-progress', 'installation-complete']:
+            # with the pages basically empty of common elements (as those are provided by the top-level wizard widget)
+            # we at least iterate over them to check this works as expected
             b.click("#" + page)
-            b.wait_js_cond(f'window.location.hash === "#/{page}"')
```

Comment:
I would implement a URL change when the next button or the wizard nav is clicked. In this way when a user refreshes they will not be landed in the main intro page (language) but they can maintain their position in the wizard.

Reply:
Good point, but seems like something for a followup PR ? Also any pointers how this can be done ?

Reply:
Sure, let's handle that in follow up. Probably activeStep property.